specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -506,6 +649,203 @@
           }
         }
       }
+    },
+    "/weather/tropical/storms/active/{format}": {
+      "get": {
+        "description": "**Get Active Storms**\n\n**Applies to**: S0 and S1 pricing tiers.\n\nGet all government-issued active storms.",
```

Comment:
Have we confirmed the SKU with PM?

Reply:
This is still under discussion

Reply:
PM has confirmed it's G1 and G2 in group chat on Aug 23.